lib/throttle.js

```diff
@@ -25,6 +25,21 @@ module.exports = (api) => {
     };
   };
 
+  api.metasync.debounce = (
+    timeout,
+    fn,
+    args
+  ) => {
+    let timer = null;
+    return () => {
+      clearTimeout(timer);
+      timer = setTimeout(
+        () => (args !== undefined ? fn(...args) : fn()),
```

Comment:
But this leads to unnesesary array instance creation. 

Do you consider line 37 the one that create an obsticle for PR to be approved?

Maybe it will be ok if I use the style `throttle` function is written with:
```
 if (args) fn(...args);
 else fn();
```